protostar/argument_parser/arg_type_test.py

```diff
@@ -38,3 +39,10 @@ def test_regex_arg_type():
     result = map_type_name_to_parser("regexp")(".*")
 
     assert isinstance(result, Pattern)
+
+
+def test_address_arg_type():
+    parser = map_protostar_type_name_to_parser("address")
+    result = parser("0xdeadbeef123")
+    assert isinstance(result, int)
+    assert result == 15302363377955
```

Comment:
This test should be in the `cli` module in the `protostar_arg_type_test.py` file. `protostar.cli` depends on `protostar.argument_parser`.